instagram/src/App.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
+import "./App.css";
+import PostsPage from "./components/PostContainer/PostsPage.js";
+import LoginPage from "./components/Login/LoginPage.js";
+import authenticate from "./components/authentication/authenticate.js";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = { loggedIn: false, username: "" };
+  }
+
+  updateState = (state, stateValue) => {
+    this.setState({ [state]: stateValue });
+  };
+
```

Comment:
great job on this. You have pretty much everything broken out into components instead of having a massive app.js file. Way to go